docs/terms.rst

```diff
@@ -23,6 +23,9 @@ An identifier can be:
 
 Remark: for any regular identifier ``i``, ``{|i|}`` is equivalent to ``i``.
 
+* an escaped notation identifier, that is, an identifier previously declared as notation,
+  wrapped in parentheses (this allows access to the notationless value of the identifier)
+
```

Comment:
Even though the system understands it as a term, I think the user should understand it as an identifier with a decorator similar to `@`.
It is only understood as a term because if it were as an identifier there would be conflicts, but the behaviour pretty much tries to reduce it as just an identifier